x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -48960,6 +48960,10 @@
     "xpack.upgradeAssistant.overview.systemIndices.statusTableColumn": "Statut",
     "xpack.upgradeAssistant.overview.systemIndices.title": "Migrer les index système",
     "xpack.upgradeAssistant.overview.systemIndices.viewSystemIndicesStatus": "Afficher les détails de migration",
+    "xpack.upgradeAssistant.overview.systemIndices.confirmModal.title": "Indice de migration",
+    "xpack.upgradeAssistant.overview.systemIndices.confirmModal.cancelButton.label": "Annuler",
+    "xpack.upgradeAssistant.overview.systemIndices.confirmModa.confirmButton.label": "Ok",
+    "xpack.upgradeAssistant.overview.systemIndices.confirmModa.description": "La migration des indices du système peut entraîner des temps d'arrêt pendant que les indices sont réindexés.Êtes-vous sûr de vouloir migrer les indices du système ?",
```

Comment:
Where did you get these translations from? We have internal services that will manually fill these, so these shouldn't be necessary

Reply:
Thank for your reminder, I have removed the relevant translations